roles/nginxplus/files/conf/http/cdh_geniza_prod.conf

```diff
@@ -46,6 +46,16 @@ server {
         proxy_set_header X-Real-IP $remote_addr;
         proxy_cache geniza_prodcache;
         health_check interval=10 fails=3 passes=2;
+
+        # allow princeton network
+        allow 128.112.0.0/16;
+        allow 140.180.0.0/16;
+        allow 172.20.95.0/24;
+        allow 172.20.192.0/19;
+        # allow access from old PGP site on cPanel
+        allow 198.199.71.236;
```

Comment:
you want this vaulted?

Reply:
should it be? doesn't seem like a big risk to me (even though I'm sure a lot of sites run on reclaim)

it's only being restricted because the production site is in development and we don't want people to look at it or use it yet